org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/dom/DOMNode.java

```diff
@@ -282,6 +282,15 @@ public DOMAttr findAttrAt(DOMNode node, int offset) {
 	}
 
 	public DOMAttr getAttributeNode(String name) {
+		return getAttributeNode(null, name);
+	}
+
+	public DOMAttr getAttributeNode(String prefix, String suffix) {

```

Comment:
use StringBuilder for string concatenation, and move that bit of code after L297